### PR TITLE
Added .editorconfig file and fixed bug with tab

### PR DESCRIPTION
**Pull request by utopiaeh**
Added .editorconfig file and fixed bug with tab on prettier and eslint after format

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_size = 4
```
Comment (cristian-sandu) [inline .editorconfig]: the preferred indent size should be 2 spaces, not 4, could you change it please ?

File: .eslintrc [hunk 2]
```diff
@@ -25,6 +24,12 @@
...
   "rules": {
     "prettier/prettier": "error",
-    "react/jsx-filename-extension": [0]
+    "react/jsx-filename-extension": [0],
+    // Indent with 4 spaces
+    "indent": ["error", 4],
+    // Indent JSX with 4 spaces
+    "react/jsx-indent": ["error", 4],
+    // Indent props with 4 spaces
+    "react/jsx-indent-props": ["error", 4]
```
Comment (cristian-sandu) [inline .eslintrc]: please change the above indents to 2 as well

Comment (cristian-sandu) [reply .eslintrc]: 4 spaces might be used for css, not for js

File: .eslintrc [hunk 3]
```diff
@@ -5,7 +5,6 @@
   },
   "extends": [
     "airbnb",
-    "prettier",
```
[the remark this answers is not in the document]
Comment (UtopiaEH) [reply .eslintrc]: by mistake, i restored now i need create a new request for this branch ?

File: .editorconfig (hunk 1 above)
Comment (UtopiaEH) [reply .editorconfig]: resolved

**Pull request merged**